static/src/stylesheets/module/crosswords/_clues.scss

```diff
@@ -36,12 +38,8 @@
     color: colour(neutral-2);
 }
 
-.crossword__clue--display-group-order {
-
-    padding-left: 0px;
-
-    //For grouped clues we display 11.12,23 accross instead of just a number
-    &:before {
-        position: initial;
-    }
+//For grouped clues we display 11.12,23 accross instead of just a number
+.crossword__clue--display-group-order:before {
+    padding-right: .875em; // 14px - Matches the width between double digit numbers and clue of non-grouped clues
```

Comment:
Yup, 14px was this width 
![image](https://cloud.githubusercontent.com/assets/638051/9872931/651f4cee-5b94-11e5-9b51-7ec0f857406e.png)

Bit of a magic number, but stylistically matches better. I also used em because the padding and min-width values were also ems so it would scale proportionately to the li font-size, as opposed to the document - what do you reckon, leave as em or just use the explicit px width (which would convert to rem)?